README.md

```diff
@@ -1,70 +1,182 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy up and running, follow these steps.
 
-### Advanced Configuration
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In order to run this project you need:
+
+- Visual Studio Code installed
+- Npm installed
+- A GitHub account
+- Git installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone git@github.com:bohaz/math-magicians.git
+
+### Install
+
+
+Install this project with:
+
+-Install Node Modules
+
+```sh
+npm install
+```
+
+### Usage
+
+To run this project, execute the following command:
+
+-Start the development server
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following commands:
+
+- ESLint linter errors run:
+
+``` npx eslint "**/*.{js,jsx}" ```
+
+-CSS linter errors run:
+
+``npx stylelint "**/*.{css,scss}"``
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using: 
+
+- Github
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ricardo Martínez**
+
+- GitHub: [@bohaz](https://github.com/bohaz)
+- Twitter: [@Ricardo29115571](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Functionality]**
+- [ ] **[Content]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/henask12/JavaScript-Capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project let me know with a STAR!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thaks Microverse for giving us the oportunity to build and contribute in projects like this. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job @bohaz 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.